### ISSUE TITLE
Does not support multiple multiple queries of same 'type'

### ISSUE THREAD

**Issue by utdrmac** [akopytov/sysbench#299]
If you look at oltp_common.lua, `point_selects` are defined as 1 single query. I want to add more of this "type". If I add another dict to the `point_selects` table, the code does not recognize this addition. The function `prepare_for_each_table` does not take in to account multiple queries inside the same stmt_defs dict. The function only loops over the number of sbtest* tables, and not over the number of queries.

Here is an example of what I think I should be able to accomplish:

```
stmt_defs = {
  my_point_selects = {
        {"SELECT * FROM imdb.title WHERE id = ?", t.INT},
        {"SELECT * FROM imdb.name WHERE id = ?", t.INT},
        {"SELECT * FROM imdb.char_name WHERE id = ?", t.INT},
        {"SELECT * FROM imdb.favorites WHERE user_id = ? AND type='actor'", t.INT},
        {"SELECT * FROM imdb.favorites WHERE user_id = ? AND type='movie'", t.INT},
        {"SELECT * FROM imdb.person_info WHERE person_id = ?", t.INT},
        {"SELECT * FROM imdb.movie_info WHERE movie_id = ?", t.INT},
        {"SELECT user2 FROM imdb.user_friends WHERE user1 = ?", t.INT}
  }
}
```

**Comment by akopytov** [comment]
I'm not sure what is the sysbench bug or even the question here. The relevant sysbench code does what I want it to do. Are you asking how you can change the code to do what you want, or I miss something?

**Comment by utdrmac** [comment]
It was my understanding that sysbench was expandable to allow others to write/add their own queries to point selects and such. The "bug" is that if I attempt to add additional queries to the `point_select` map/table, it won't work. The existing code in oltp_common assumes only 1 query per "type".

**Comment by akopytov** [comment]
sysbench is expandable, because arbitrary workloads can be defined by users in a Turing complete language.

It also contains a few bundled workloads that have existed for a long time. Users are free to either use them, or create their own ones, possibly using existing workloads as a base or example as they see fit. Changing existing code to accommodate it for every single customized workload is precisely the situation I wanted to avoid by introducing Lua scripting.

In other words, copy `oltp_point_select.lua` to `mybench.lua`. Also copy `oltp_common.lua` to `mybench_common.lua`. Change them as necessary to do what you need. Share it with others on Github or SysbenchRocks.

**Comment by utdrmac** [comment]
I was hoping the oltp_common.lua would have been more generic. My `mybench_common.lua` has a lot of repeated code from oltp_common.
